chicken_app/src/Pages/Signup.js

```diff
@@ -91,7 +95,7 @@ function Signup(props) {
         await signup(emailRef.current.value, passwordRef.current.value)
         navigate("/profile/confirm-email")
       } catch {
-        setAuthError({ isAuthError:true, errorDetails:"creat acct error"})
+        setAuthError({ isAuthError:true, errorDetails:"Email is already in use"})
```

Comment:
Good job on making it so that the form doesn't submit if the email is invalid. Can you also change this error text so that it says that the email is invalid instead of saying that the email is already being used when someone is entering an invalid email?

Reply:
firebase will give some errors, one will be email already in use- based on which error we get, we will display different details

Reply:
It's saying that the email is already in use when the email is just improperly formatted.